### ISSUE TITLE
multiscale implementation

### ISSUE THREAD

**Issue by yformer** [gaopengcuhk/SMCA-DETR#8]
Hi, I checked the implementation. But I did not find the code implement the multiscale part. Would you like to add it?

**Comment by gaopengcuhk** [comment]
Please check Multi_scale_SMCA.zip. By the way, we will release a pre-trained model of Multi-scale smca accompanied with panoptic segmentation in the future.

**Comment by yformer** [comment]
Thanks. Another question for this snippet code.
`
        if isinstance(samples[0], (list, torch.Tensor)):
            samples[0] = nested_tensor_from_tensor_list(samples[0])
        features, pos = self.backbone(samples[0])
        h_w = torch.stack([torch.stack([inst['size'] for inst in samples[1]])[:, 1], torch.stack([inst['size'] for inst in samples[1]])[:, 0]], dim=-1)
`
The input samples is a NestedTensor. How can you access it by its index, such as samples[0]? And what is samples[1]? For h_w, are you trying to get [[h, w], ..., [h, w]]?
